.github/workflows/test-add-compiler.yaml

```diff
@@ -1,69 +1,59 @@
 ---
-name: "Add Compiler test"
-
+name: Add Compiler test
 on:
   workflow_dispatch:
     inputs:
       image:
-        description: 'GCP image for test cluster'
```

Comment:
The quoting changes are quite confusing and make the diffs huge. I already mentioned that in another PR when single quotes we converted to double quotes, but only in some files. Is the change here to no quotes really required?

Reply:
Hey @bastelfreak, we are in the process to convert all the action YAML files to use the same formatting. So take this as the current format. There will be another pr to convert all other files to the same format.